src/util/getCircleCoords.js

```diff
@@ -0,0 +1,20 @@
+import external from '../externalModules';
+
+/**
+ * Retrieve the bounds of the circle in image coordinates
+ *
+ * @param {*} startHandle
+ * @param {*} endHandle
+ * @returns {{ left: number, top: number, width: number, height: number }}
+ */
+export default function getCircleCoords(startHandle, endHandle) {
+  const { distance } = external.cornerstoneMath.point;
+  const radius = distance(startHandle, endHandle);
+
+  return {
+    left: Math.round(Math.min(startHandle.x - radius, endHandle.x)),
+    top: Math.round(Math.min(startHandle.y - radius, endHandle.y)),
```

Comment:
Why is rounding "safe" here? Any potential pitfalls?

Reply:
We have a method that finds a circles bounding box. This doesn't feel a ton different.